Jenkinsfile

```diff
@@ -6,8 +6,8 @@ properties([
                               artifactNumToKeepStr: '',
                               daysToKeepStr: '',
                               numToKeepStr: '100')),
-    parameters([string(name: 'RSTUDIO_VERSION_MAJOR', defaultValue: '1', description: 'RStudio Major Version'),
-                string(name: 'RSTUDIO_VERSION_MINOR', defaultValue: '5', description: 'RStudio Minor Version'),
+    parameters([string(name: 'RSTUDIO_VERSION_MAJOR', defaultValue: '2021-07', description: 'RStudio Major Version'),
```

Comment:
One worry I have here is that some existing code might be assuming that the major version component is a parsable number; this change will break that assumption.

Is having the year as the major version, and month as the minor version, a suitable compromise?

Reply:
I don't believe the scheme for the version is debatable. We must have the minor version able to increment for patches (0, 1, 2, etc.). This is a change across all RStudio products, AFAIK

Reply:
Why not use the patch version component for patches?

Reply:
We don't actually have a patch version, that just ends up being build number. I also don't think we can change the versioning scheme. The major version needs to be YYYY-MM

Reply:
Gotcha. The main thing that I think needs to be validated then is that `RStudio.Version()` still produces a parsable version in the `version` field, and that `rstudioapi::getVersion()` also produces a parsable version (but I think that just reads that same field).

Reply:
Got it, I will test this

Reply:
The output:
```R
> RStudio.Version()
$mode
[1] "server"

$version
[1] ‘2021.7.0.1011’

$release_name
[1] "Ghost Orchid"

> rstudioapi::getVersion()
[1] ‘2021.7.0.1011’
```